requirements.txt

```diff
@@ -11,3 +11,4 @@ requests==2.27.1
 six==1.16.0
 urllib3==1.26.9
 wrapt==1.14.1
+
```

Comment:
If you didn't mean to add this newline, you may want to revert it to keep the history cleaner.